README.md

```diff
@@ -6,7 +6,22 @@ This is a CDK project which configures a 2-instance ASG. The ASG is pointed to a
 This is a blank project for Python development with CDK.
 
 
-### How to run
+## How to run
+
+### Preparing the AMI
+
+- Install [packer](https://www.packer.io/intro/getting-started/install.html)
+- Generate [AWS access keys](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey)
+- Set the following variables in your shell's environment: `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_DEFAULT_REGION`
+- Change into the `packer` sub directory: `cd packer`
+- Build the Folding at Home Amazon Machine Image that will be used to create the virtual machines
+      
+      packer build -var 'fah_user=your_username' -var 'fah_passkey=your_passkey' \
+                  -var 'fah_team=your_team_id' fah_ami.json
+
+- From the output of the above command, copy the generated AMI's Id for use later
```

Comment:
ideas for improvement: use the ami id by using the [manifest postprocessor](https://www.packer.io/docs/post-processors/manifest/) and auto-populate config.yaml

Reply:
made to feature #5

Reply:
This totally makes sense, let's do this in a follow up PR :)